corehq/apps/reports/models.py

```diff
@@ -768,21 +781,19 @@ def _get_and_send_report(self, language, emails):
                 else self.email_subject
             )
             attach_excel = getattr(self, 'attach_excel', False)
-            body, excel_files = get_scheduled_report_response(
-                self.owner, self.domain, self._id, attach_excel=attach_excel
-            )
             for email in emails:
-                send_html_email_async.delay(title, email, body.content,
-                                            email_from=settings.DEFAULT_FROM_EMAIL,
-                                            file_attachments=excel_files, ga_track=True,
-                                            ga_tracking_info={'project_space_id': self.domain})
+                body, excel_files = get_scheduled_report_response(
+                    self.owner, self.domain, self._id, attach_excel=attach_excel, user_email=email)
```

Comment:
You should take a similar approach to below where you render the body outside the loop and then only render the full content inside the loop